### PR TITLE
CORE-15149 Remove the old contract testing API

### PR DESCRIPTION
**Pull request by nkovacsx**
### Overview

We have a separate repo for contract testing now: https://github.com/corda/corda5-contract-testing
We should use that as a dependency rather than recreating our own.

This work was initially done by @jennyang-r3, I just merged with `release/os/1.0`.

### CODE REVIEW

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
See comment

File: build.gradle [hunk 1]
```diff
@@ -91,7 +91,9 @@ subprojects {
 
     repositories {
         mavenLocal()
-
+        maven {
+            url = "https://download.corda.net/maven/corda-dependencies"
+        }
```
Comment (ronanbrowne) [inline build.gradle]: Why do we need this new repository declared here?

I only see `com.r3.corda.ledger.utxo:contract-testin` added

which is not in `corda-dependencies`  artifactory location

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
See comment

File: build.gradle (hunk 1 above)
Comment (nkovacsx) [reply build.gradle]: I picked these changes from Jenny's old branch so probably just left that in there by mistake.
Removing now.

**Review by ronanbrowne — DISMISSED** [review]
LGTM (regarding my previous comments)

File: gradle.properties [hunk 2]
```diff
@@ -18,6 +18,9 @@ cordaNotaryPluginsVersion=5.2.0.0-beta+
 # Specify the version of the cordapp-cpb2 and cordapp-cpk2 plugins
 cordaPluginsVersion=7.0.3
```
Comment (ronanbrowne) [inline gradle.properties]: Actually, one more point, I noticed after hitting approved.

 `7.0.4` has been released now for a while and is what runtime 5.2 [uses](https://github.com/corda/corda-runtime-os/blob/release/os/5.2/gradle.properties#L27), perhaps this should be in line here also

**Review by blsemo — APPROVED** [review]
LGTM

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Pull request merged**